package.json

```diff
@@ -15,6 +15,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@polymer/polymer": "^1.2.5-npm-test.2",
```

Comment:
Because that is the latest Polymer 1.2 package the maintainers have released. Their stable versions in npmjs.com are:

```
   [ '1.2.5-npm-test.1',
     '1.2.5-npm-test.2',
     '1.7.1-npm-test.3',
     '1.7.1-npm-test.4',
     '2.0.0-rc.8',
     '2.0.0',
     '2.0.1',
     '2.0.2' ],
```

Refs Polymer/polymer#3903